whitepaper/chapters/trees.tex

```diff
@@ -56,20 +56,20 @@ \subsection{Merkle Tree}
 	\nemtree{
 		\tikzset{ highlight/.style = {fill = yellow} }
 		\tikzstyle{level 3}=[sibling distance=8em]
-		\node [highlight] {Merkle Hash = $H_{Root}$ =\\H($H_{ABCD}$, $H_{EE^2}$)}
-			child [sibling distance=16em] { node {$H_{ABCD}$ =\\H($H_{AB}$, $H_{CD}$)}
-				child { node {$H_{AB}$ =\\H(H(A), H(B))}
-					child { node [highlight] {H(A)} { child { node {A} } } }
-					child { node {H(B)} { child { node [highlight] {B} } } }
+		\node [highlight] {Merkle Hash = $H_{Root}$ =\\ $\hf(H_{ABCD}, H_{EE^2})$}
+			child [sibling distance=16em] { node {$H_{ABCD}$ =\\ $\hf(H_{AB}, H_{CD})$}
+				child { node {$H_{AB}$ =\\ $\hf(\hf(A), \hf(B))$ }
+					child { node [highlight] { $\hf(A)$ } { child { node {A} } } }
+					child { node { $\hf(B)$ } { child { node [highlight] {B} } } }
 				}
-				child { node [highlight] {$H_{CD}$ =\\H(H(C), H(D))}
-					child { node {H(C)} { child { node {C} } } }
-					child { node {H(D)} { child { node {D} } } }
+				child { node [highlight] {$H_{CD}$ =\\ $\hf(\hf(C), \hf(D))$ }
+					child { node { $\hf(C)$ } { child { node {C} } } }
+					child { node { $\hf(D)$ } { child { node {D} } } }
 				}
 			}
-			child [sibling distance=16em] { node [highlight] {$H_{EE^2}$ =\\H($H_{EE}$, $H_{EE}$)}
-				child { node {$H_{EE}$ =\\H(H(E), H(E))}
-					child { node {H(E)} { child { node {E} } } }
+			child [sibling distance=16em] { node [highlight] {$H_{EE^2}$ =\\ $\hf(H_{EE}, H_{EE})$}
+				child { node {$H_{EE}$ =\\ }
```

Comment:
did you intentionally remove?

Reply:
couldn't figure out what you're referring to, now I see it, no fixed